python/setup.cfg

```diff
@@ -78,6 +79,7 @@ doc =
     xgboost>=1.3.0
     lightgbm>=3.3.5
     matplotlib
+    datasets
```

Comment:
Add a dataset extra requirements.

Reply:
This is for sphinx to get correct source module. The `dataset` extra requirement is already there.